ipynb/scripts/make-representative-sequences.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+
+mport qiime2 as q2
+from biom import Table
+bt = q2.Artifact.load('deblur-ag/table.qza').view(Table)
```

Comment:
I think we don't need to do this, as this file will be generated by the commands in the notebook.